openest/test/curves/test_bounding.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+
+from openest.curves.bounding import *
```

Comment:
It's in PEP8 but on principle, I'd avoid wildcard imports, especially on tests. It makes it unclear what's actually in the namespace, what's being tested/used as dependency, etc.